src/main/java/com/welcome/bot/repository/InviteRepository.java

```diff
@@ -6,12 +6,13 @@
 import org.springframework.stereotype.Repository;
 
 import com.welcome.bot.domain.Invite;
-import com.welcome.bot.domain.UserSettings;
+
+
 
 @Repository
 public interface InviteRepository extends JpaRepository<Invite, Long> {
 
-    Optional<Invite> findById(Long id);
+    Optional<Invite> findById(String id);
```

Comment:
Why string as argument? Id is Long type.

Reply:
Resolved.